chainer/functions/loss/softmax_cross_entropy.py

```diff
@@ -109,22 +109,19 @@ def _is_chainerx_supported(self, input_arrays):
         return True
 
     def forward_chainerx(self, inputs):
-        x, t = inputs
-        n_classes = x.shape[1]
-        score = chainerx.log_softmax(x, axis=1)
-        mask = (t[:, chainerx.newaxis] == chainerx.arange(
-            n_classes, dtype=t.dtype, device=x.device)).astype(score.dtype)
+        if self.reduce == 'mean' and self.normalize:
+            x, t = inputs
+            n_classes = x.shape[1]
+            score = chainerx.log_softmax(x, axis=1)
+            mask = (t[:, chainerx.newaxis] == chainerx.arange(
+                n_classes, dtype=t.dtype, device=x.device)).astype(score.dtype)
+            y = -(score * mask).sum() * (1 / mask.sum())
```

Comment:
One less routine call.
```suggestion
            y = (score * mask).sum() * (-1 / mask.sum())
```

Reply:
Nice!